### DISCUSSION TITLE
Resize Lora functionality

### DISCUSSION THREAD

**Discussion by ManglerFTW** [bmaltais/kohya_ss#796] [Q&A]
Does anyone know where I can find documentation on the resize lora functionality? How does it work? For instance. If I have one lora that is a network rank of 64 and another that is 128, is it possible to resize one of them using this so that they can be merged?

**Comment by TomLucidor** [comment]
I have similar ideas here as well https://github.com/bmaltais/kohya_ss/discussions/1751

My guess:
- The largest network rank is the right size (if all are competently sized and the largest one has the most info)
- The median/mode network rank (assuming most having standard quality and the outliers are the exception)
- Might need to double or quadruple the size if there are dozens of LoRAs
- 8 is for testing, 16 is small, 32 is medium, 64 is large, and 128 feels a bit brute-forced
